### PR TITLE
Document enabling New Relic on Platform.sh

### PR DESCRIPTION
**Pull request by GuGuss**

### CODE REVIEW

File: src/administration/integrations/new-relic.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Enable the New Relic extension in your `.platform.app.yaml` as follows:
+
+```bash
+runtime:
+    extensions:
+        - newrelic
+```
+
+Push the changes to your Platform environment to enable Blackfire as follows:
```
Comment (ralt) [inline src/administration/integrations/new-relic.md]: Blackfire? 🙂

Comment (ralt) [inline src/administration/integrations/new-relic.md]: `s/Platform/Platform.sh/`

File: src/administration/integrations/new-relic.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+That's it! You need to wait a little bit for your New Relic dashboard to be generated.
+
+## Troubleshoot
+
+Additionally, you can check that your application is properly connected to New Relic by looking at the application logs:
+
+```bash
+platform ssh
```
Comment (pjcdawkins) [inline src/administration/integrations/new-relic.md]: How about `platform log app` ?

Comment (GuGuss) [inline src/administration/integrations/new-relic.md]: I didn't know about that command 👍

**Comment by ralt** [comment]
Should it be mentioned that only PHP supports New Relic atm?

**Comment by christopher-hopper** [comment]
I'm looking for some docs on how to add a NewRelic Deployment API call to my `.platform.app.yml`. For the most part it's simple, just one thing I don't quite have yet though is how best to locate the following vairables for use in a call like this:

- `$GIT_SHA`
- `$GIT_LOG`
- `$GIT_USER`
- `$PLATFORM_ENVIRONMENT`

```
curl -X POST 'https://api.newrelic.com/v2/applications/123456789/deployments.json' \
     -H 'X-Api-Key:XXXXXXX42890e8ea4XXXXXXXa612b81dXXXXXXc3XXX' -i \
     -H 'Content-Type: application/json' \
     -d \
'{
  "deployment": {
    "revision": "$GIT_SHA",
    "changelog": "$GIT_LOG",
    "description": "Deployment to $PLATFORM_ENVIRONMENT",
    "user": "$GIT_USER"
  }
}' 
```

If Git is available to me, on both Platform.sh and Platform.sh Enterprise, during the deploy hook run then I'll be fine, except for the `$PLATFORM_ENVIRONMENT` variable. I'll take a look and see if that's available, I know I'll have the `$PLATFORM_BRANCH` but not sure about the `$PLATFORM_ENVIRONMENT`.

**Comment by christopher-hopper** [comment]
:open_mouth: I found something helpful. Okay now we're getting somewhere.

So, the list of environment variables provided by Platform.sh is buried here:

- [Platform.sh-provided variables](https://docs.platform.sh/development/variables.html#platformsh-provided-variables)

The NewRelic API documentation for a Deployment is buried here:

- [NewRelic API > Application Deployments > Create](https://rpm.newrelic.com/api/explore/application_deployments/create)

This would be well worth documenting as an example in the Docs you're proposing. I can add it as a PR once this is merged, or we can do it now.

```yml
hooks:
    # We run deploy hook after your application has been deployed and started.
    # Deploy hooks can access services, but the file system is now read-only.
    deploy: |
        if [ "${PLATFORM_BRANCH}" = "master" -a -x /usr/bin/curl ]; then
          /usr/bin/curl -X POST 'https://api.newrelic.com/v2/applications/123456789/deployments.json' \
               -H 'X-Api-Key:XXXXXXX42890e8ea4XXXXXXXa612b81dXXXXXXc3XXX' -i \
               -H 'Content-Type: application/json' \
               -d \
          '{
            "deployment": {
              "revision": "$PLATFORM_TREE_ID",
              "description": "Project: ${PLATFORM_PROJECT}, Environment: ${PLATFORM_ENVIRONMENT}, Branch: ${PLATFORM_BRANCH}",
            }
          }'
        fi
```

**Comment by Crell** [comment]
@christopher-hopper That seems overly complex.  We only just recently added New Relic APM support in PS.  Can you check the documentation in this PR and see if it works for you?  If so then we can go ahead and merge it.

**Comment by xtfer** [comment]
This looks okay to me and ready to merge.

**Pull request merged**